docs/spec/staking/spec-technical.md

```diff
@@ -99,11 +118,8 @@ type Description struct {
 ```
 
 Candidate parameters are described:
- - Status: signal that the candidate is either vying for validator status,
-   either unbonded or unbonding, an active validator, or a kicked validator
-   either unbonding or unbonded.
- - PubKey: separated key from the owner of the candidate as is used strictly
-   for participating in consensus.
+ - Status: it can be Bonded (active validator), Unbonded (validator candidate) or Revoked
+ - PubKey: candidate public key that is used strictly for participating in consensus
```

Comment:
The PubKey is also used for identification when delegating atoms (you delegate to a PubKey)